pkg/daemon/node.go

```diff
@@ -51,7 +54,15 @@ func getNodeAnnotation(client corev1.NodeInterface, node string, k string) (stri
 
 // getNodeAnnotationExt is like getNodeAnnotation, but allows one to customize ENOENT handling
 func getNodeAnnotationExt(client corev1.NodeInterface, node string, k string, allowNoent bool) (string, error) {
-	n, err := client.Get(node, metav1.GetOptions{})
+	var n *core_v1.Node
+	err := wait.PollImmediate(10*time.Second, 5*time.Minute, func() (bool, error) {
+		var err error
+		n, err = client.Get(node, metav1.GetOptions{})
+		if err == nil {
+			return true, nil
+		}
+		return false, nil
```

Comment:
Shouldn't we return the error here? (We also didn't do it in the loop in `getMachineConfig`)

Reply:
Ah, right, sorry. I think it's fine as is for now then.